src/tasks/tasks.conroller.js

```diff
@@ -25,29 +25,33 @@ tasks.get('/', (req, res) => {
     }
 })
 
-tasks.post('/', validateBody, (req, res) => {
+tasks.post('/', validateBody, async (req, res) => {
     const task = req.body
-    const obj = save(task)
-    res.status(201)
-    res.send(obj)
+    try {
+        const obj = await saveTask(task)
+        res.status(201)
+        res.send(obj)
+    } catch (error) {
+        throw new ErrorHandler(404, "Task does't save(")
+    }  
 })
 
-tasks.delete('/:id', (req, res) => {
+tasks.delete('/:id', async (req, res) => {
     try {
         const id = req.params.id
-        const obj = del(parseInt(id))
+        await delTask(parseInt(id))
         res.status(200)
-        res.send(obj)
+        res.send("This elem has been deleted, you can't backup his.")
```

Comment:
return to the client deleted entity when set status 200
OR
doesn't provide removed entity and set status 204 (No Content)